test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -926,7 +926,9 @@ UCS_TEST_P(test_ucp_sockaddr, ep_query) {
 
 UCS_TEST_P(test_ucp_sockaddr, set_local_sockaddr)
 {
-    if (m_test_addr.is_rdmacm_netdev()) {
+    if (!m_test_addr.is_rdmacm_netdev()) {
+        UCS_TEST_SKIP_R("skip non-rdmacm devices");
```

Comment:
still not clear what's the actual reason

Reply:
removed if